src/sider/mote/components/endre/EndreDialogmoteContainer.tsx

```diff
@@ -1,17 +1,73 @@
-import { DialogmoteSideContainer } from "../../../../components/dialogmote/DialogmoteSideContainer";
 import EndreDialogmoteSkjema from "./EndreDialogmoteSkjema";
+import { useParams } from "react-router-dom";
+import { useDialogmoterQuery } from "@/data/dialogmote/dialogmoteQueryHooks";
+import { useBrukerinfoQuery } from "@/data/navbruker/navbrukerQueryHooks";
+import Side from "@/sider/Side";
+import { Menypunkter } from "@/components/globalnavigasjon/GlobalNavigasjon";
+import SideLaster from "@/components/SideLaster";
+import Sidetopp from "@/components/Sidetopp";
+import { BrukerKanIkkeVarslesPapirpostAdvarsel } from "@/components/dialogmote/BrukerKanIkkeVarslesPapirpostAdvarsel";
+import Feilmelding from "@/components/Feilmelding";
+import * as Tredelt from "@/sider/TredeltSide";
+import { DeltakereSvarInfo } from "@/components/dialogmote/DeltakereSvarInfo";
+import { BodyLong, Box, Heading } from "@navikt/ds-react";
+import { MoteIkonBlaaImage } from "../../../../../img/ImageComponents";
 
 const texts = {
   pageTitle: "Endre dialogmøte",
   pageHeader: "Endre dialogmote",
+  moteNotFound: "Fant ikke dialogmøte",
+  motesvarHeading: "Møtesvar",
+  motesvarSubheading: "Viser svar på siste innkalling/endring",
 };
 
 const EndreDialogmoteContainer = () => {
+  const { dialogmoteUuid } = useParams<{
+    dialogmoteUuid: string;
+  }>();
+  const { isLoading, isError, data: dialogmoter } = useDialogmoterQuery();
+  const { brukerKanIkkeVarslesDigitalt } = useBrukerinfoQuery();
+
+  const dialogmote = dialogmoter.find(
+    (dialogmote) => dialogmote.uuid === dialogmoteUuid
+  );
+
   return (
-    <DialogmoteSideContainer title={texts.pageTitle} header={texts.pageHeader}>
-      {(dialogmote) => <EndreDialogmoteSkjema dialogmote={dialogmote} />}
-    </DialogmoteSideContainer>
+    <Side tittel={texts.pageTitle} aktivtMenypunkt={Menypunkter.DIALOGMOTE}>
+      <SideLaster henter={isLoading} hentingFeilet={isError}>
+        <Sidetopp tittel={texts.pageHeader} />
+        {dialogmote ? (
+          <Tredelt.Container>
+            <Tredelt.FirstColumn>
+              {brukerKanIkkeVarslesDigitalt && (
+                <BrukerKanIkkeVarslesPapirpostAdvarsel />
+              )}
+              <EndreDialogmoteSkjema dialogmote={dialogmote} />
+            </Tredelt.FirstColumn>
+            <Tredelt.SecondColumn>
+              <Box
+                background="surface-default"
+                className="flex flex-col gap-8 p-6"
+              >
+                <div className="flex gap-4">
+                  <img src={MoteIkonBlaaImage} alt="moteikon" />
+                  <div>
+                    <Heading level="2" size="medium">
+                      {texts.motesvarHeading}
+                    </Heading>
+                    <BodyLong size="small">{texts.motesvarSubheading}</BodyLong>
+                  </div>
+                </div>
+                <DeltakereSvarInfo dialogmote={dialogmote} />
+              </Box>
```

Comment:
Burde ikke dette vært en egen komponent, siden den brukes flere steder? Eller var det det du nevnte i kommentaren over?

Reply:
Brukes ikke noen flere steder per nå, men som nevnt over så kan det hende vi kan lage en ny DialogmoteSideContainer med støtte for tredeling senere hvis vi f.eks. ønsker det på Avlys-siden også,

Reply:
Åja, jeg trodde denne komponenten også var på "forsiden" av dialogmøte-siden, ikke bare inne på "Endre møte"?

Reply:
Ah, ja `<DeltakereSvarInfo>` brukes der, men uten ikon og heading over som det er lagt på her.
![image](https://github.com/navikt/syfomodiaperson/assets/79838644/1451b704-c02b-4e2d-a99e-e4db3a5748c9)

Reply:
Aha my bad!